kolibri/core/tasks/registry.py

```diff
@@ -157,7 +157,34 @@ def __init__(
         cancellable=False,
         track_progress=False,
         permission_classes=None,
+        long_running=False,
     ):
+        """
+        :param func: Function to be wrapped as a Registered task
+        :type func: function
+        :param job_id: Fixed job_id to use for any job run from this task, defaults to None
+        :type job_id: str
+        :param queue: The queue to run this task in by default, defaults to DEFAULT_QUEUE
+        :type queue: str
+        :param validator: The job validator, used to validate and serialize JSON into
+        the args, kwargs, and job arguments for running a job of this task,
+        defaults to JobValidator
+        :type validator: JobValidator
+        :param priority: The priority for this task, can be either HIGH or REGULAR,
+        defaults to Priority.REGULAR
+        :type priority: int
+        :param cancellable: Can this task be cancelled while running, defaults to False
+        :type cancellable: bool
+        :param track_progress: Does this task track progress while it's executing, defaults to False
+        :type track_progress: bool
+        :param permission_classes: Classes that determine if a user has read and write access to
+        jobs of this task, defaults to None
+        :type permission_classes: list of BasePermission derived classes or
+        instantiated BasePermission derived classes.
+        :param long_running: In regular operation should this task finish in under ten minutes,
```

Comment:
where is this limit implemented? Looking at the Jobs class I don't find it

Reply:
This is purely an advisory flag at the moment - the main reason for having it is that on Android if a task is going to run for longer than ten minutes, we need to tell the OS and foreground it, otherwise it risks being killed.

I had considered allowing this to be a callable rather than a boolean, but it seemed simplest to implement it as a boolean first.